components/Carousel/Carousel.js

```diff
@@ -1,5 +1,42 @@
 class Carousel {
+  constructor(element) {
+    this.element = element;
 
+    // Puts Carousel Items into an array
+    this.items = element.querySelectorAll(".Carousel__item");
+    this.items = Array.from(this.items);
+    console.log(this.items);
+    this.index = 0;
+
+    // Adds arrow elements into an array
+    this.arrows = element.querySelectorAll(".Arrow");
```

Comment:
Different implementations than the ones I have seen, but no worries this is great. This goes to show that there are many ways of accomplishing things, just find the one you feel most comfortable with.